### PR TITLE
chore: stylua column width 120 -> 140

### PR DESCRIPTION
**Pull request by alex-courtis**
No `"if$"` remaining, adds readability.

#2449 increases to 160 however those long lines are genuinely too long. Manually resolved those, see with comments.

### CODE REVIEW

File: lua/nvim-tree/watcher.lua [hunk 1]
```diff
@@ -73,14 +73,10 @@ function Event:start()
 
   rc, _, name = self._fs_event:start(self._path, FS_EVENT_FLAGS, event_cb)
   if rc ~= 0 then
-    local warning = string.format("Could not start the fs_event watcher for path %s : %s", self._path, name)
     if name == "EMFILE" then
-      M.disable_watchers(
-        warning,
-        "Please see https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#could-not-start-fs_event-for-path--emfile"
-      )
+      M.disable_watchers "fs.inotify.max_user_watches exceeded, see https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting"
```
Comment (alex-courtis) [inline lua/nvim-tree/watcher.lua:77]: Tidied message.

File: lua/nvim-tree/diagnostics.lua [hunk 2]
```diff
@@ -95,11 +95,9 @@ function M.update()
...
           local nodepath = utils.canonical_path(node.absolute_path)
           log.line("diagnostics", "  %d checking nodepath '%s'", line, nodepath)
-          if
-            M.show_on_dirs
-            and vim.startswith(bufpath:gsub("\\", "/"), nodepath:gsub("\\", "/") .. "/")
-            and (not node.open or M.show_on_open_dirs)
-          then
+
+          local node_contains_buf = vim.startswith(bufpath:gsub("\\", "/"), nodepath:gsub("\\", "/") .. "/")
+          if M.show_on_dirs and node_contains_buf and (not node.open or M.show_on_open_dirs) then
```
Comment (alex-courtis) [inline lua/nvim-tree/diagnostics.lua:100]: Broke up.

File: lua/nvim-tree/renderer/components/git.lua [hunk 3]
```diff
@@ -94,11 +94,7 @@ end
 local function nil_() end
 
 local function warn_status(git_status)
-  notify.warn(
-    'Unrecognized git state "'
-      .. git_status
-      .. '". Please open up an issue on https://github.com/nvim-tree/nvim-tree.lua/issues with this message.'
-  )
+  notify.warn(string.format("Unrecognized git state '%s'", git_status))
```
Comment (alex-courtis) [inline lua/nvim-tree/renderer/components/git.lua:97]: Message is no longer needed; we were fishing for unknown statuses.

File: lua/nvim-tree/actions/node/open-file.lua [hunk 4]
```diff
@@ -62,13 +62,7 @@ local function pick_win_id()
...
 
   if #M.window_picker.chars < #selectable then
-    notify.error(
-      string.format(
-        "More windows (%d) than actions.open_file.window_picker.chars (%d) - please add more.",
-        #selectable,
-        #M.window_picker.chars
-      )
-    )
+    notify.error(string.format("More windows (%d) than actions.open_file.window_picker.chars (%d).", #selectable, #M.window_picker.chars))
```
Comment (alex-courtis) [inline lua/nvim-tree/actions/node/open-file.lua:65]: Action is obvious :)

File: lua/nvim-tree/actions/node/system-open.lua [hunk 5]
```diff
@@ -16,18 +16,18 @@ function M.fn(node)
...
-    { args = process.args, stdio = { nil, nil, process.stderr }, detached = true },
-    function(code)
-      process.stderr:read_stop()
-      process.stderr:close()
-      process.handle:close()
-      if code ~= 0 then
-        notify.warn(string.format("system_open failed with return code %d: %s", code, process.errors))
-      end
+
+  local opts = { args = process.args, stdio = { nil, nil, process.stderr }, detached = true }
```
Comment (alex-courtis) [inline lua/nvim-tree/actions/node/system-open.lua:20]: broke up

File: lua/nvim-tree/git/runner.lua [hunk 6]
```diff
@@ -156,13 +156,7 @@ function Runner:_finalise(opts)
...
     timeouts = timeouts + 1
     if timeouts == MAX_TIMEOUTS then
-      notify.warn(
-        string.format(
-          "%d git jobs have timed out after %dms, disabling git integration. Try increasing git.timeout",
-          timeouts,
-          opts.timeout
-        )
-      )
+      notify.warn(string.format("%d git jobs have timed out after git.timeout %dms, disabling git integration.", timeouts, opts.timeout))
```
Comment (alex-courtis) [inline lua/nvim-tree/git/runner.lua:159]: Message was too verbose

**Review by Akmadan23 — APPROVED** [review]
Overall everything looks good to me, maybe I would format a couple of tables line by line instead of all on a single line, but that's a really minor gripe.

**Comment by alex-courtis** [comment]
> Overall everything looks good to me, maybe I would format a couple of tables line by line instead of all on a single line, but that's a really minor gripe.

Please do. Single line is situational, with line by line being much more readable and the general convention.

**Comment by Akmadan23** [comment]
> @Akmadan23 can you please raise another PR?

Of course

**Pull request merged**